pkg/controller/controller_ref_manager.go

```diff
@@ -565,6 +572,7 @@ func ownerRefControllerPatch(controller metav1.Object, controllerKind schema.Gro
 					BlockOwnerDeletion: &blockOwnerDeletion,
 				},
 			},
+			Finalizers: finalizers,
```

Comment:
does this add or overwrite the whole list?

Reply:
adds, due to the patchStrategy